VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -430,14 +431,22 @@ export abstract class WaitingRoomBaseComponent {
             return;
         }
 
-        if (this.hearing.isInSession()) {
+        if (this.hearing.isInSession() && this.participant.hearing_role !== HearingRole.WITNESS) {
             this.logger.debug('Showing video because hearing is in session');
             this.showVideo = true;
             this.showConsultationControls = false;
             this.isPrivateConsultation = false;
             return;
         }
 
+        if (this.participant.hearing_role === HearingRole.WITNESS && this.participant.status === ParticipantStatus.InHearing) {
+            this.logger.debug('Showing video because witness is in hearing');
```

Comment:
When you merge master into your branch, could you double check the log message format please? Thanks

Reply:
Updated the PR, have a look